contract/io/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "channel-io"
+version = "0.1.0"
+edition = "2021"
+license = "GPL-3.0"
+authors = ["Gear Technologies"]
+
+[dependencies]
+gstd = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
+codec = { package = "parity-scale-codec", version = "3.2.1", default-features = false }
+scale-info = { version = "2.2.0", default-features = false }
```

The latest is `v2.3.0`, as far as I remember.
```suggestion
scale-info = { version = "2.3.0", default-features = false }
```